orchestration/hca_orchestration/solids/create_snapshot.py

```diff
@@ -55,7 +61,12 @@ def get_completed_snapshot_info(context: AbstractComputeExecutionContext, job_id
             EventMetadataEntry.text(snapshot_info_dict['id'], "snapshot_id",
                                     description="Snapshot ID in the data repo"),
             EventMetadataEntry.text(job_id, "job_id", description="Successful data repo job ID"),
-        ]
+        ],
+        tags={
```

Comment:
Adding these tags for easier access to the snapshot info during the test that exercises this code.